python/py-pyqt5/Portfile

```diff
@@ -41,6 +44,37 @@ if {${subport} eq "${name}-common"} {
         file copy {*}[glob -directory ${worksrcpath}/sip/ Q*] ${destroot}${prefix}/share/sip/PyQt5
         file copy ${worksrcpath}/sip/Enginio ${destroot}${prefix}/share/sip/PyQt5
     }
+
+} elseif {[string first "webengine" ${subport}] != -1} {
+    PortGroup           qmake5 1.0
+
+    description         PyQt5 Webengine bindings
+    long_description    ${description}
+    master_sites        https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/${version}
+    distname            PyQtWebEngine_gpl-${version}
+    checksums           rmd160  5e0c3cd69448b8dee042ba6565c823e266c22a14 \
+                        sha256  860704672ea1b616e1347be1f347bc1c749e64ed378370863fe209e84e9bd473 \
+                        size    42474
+
+    depends_lib-append  port:py${python.version}-pyqt5
+    qt5.depends_component \
+                        qtwebengine
+    use_configure       yes
```

Comment:
Isn't this the default? I'm not very familiar with the qmake portgroup.

Reply:
I am neither...

Reply:
I think the qmake portgroup doesn't explicitly set it, while the python portgroup explicitly sets it to `no`.

Reply:
Thank you Mark! Let's keep it in there then.